tests/decorators/test_uri_parsing.py

```diff
@@ -1,18 +1,26 @@
 import pytest
 from connexion.decorators.uri_parsing import (AlwaysMultiURIParser,
                                               FirstValueURIParser,
+                                              OpenAPIURIParser,
                                               Swagger2URIParser)
 from werkzeug.datastructures import MultiDict
 
 QUERY1 = MultiDict([("letters", "a"), ("letters", "b,c"),
                     ("letters", "d,e,f")])
 QUERY2 = MultiDict([("letters", "a"), ("letters", "b|c"),
                     ("letters", "d|e|f")])
+QUERY3 = MultiDict([("letters[eq]", ["a"]), ("letters[eq]", ["b", "c"]),
```

Comment:
What is the query string that leads to this `MultiDict`?

Reply:
Its a API Rest design named LHS Brackets : 
- https://christiangiacomi.com/posts/rest-design-principles/